src/main/java/com/treasuredata/bigdam/log/Log.java

```diff
@@ -95,9 +94,9 @@ public static Logger defaultLoggerGetter(final Class<?> clazz)
         return LoggerFactory.getLogger(clazz);
     }
 
-    public static SentryClient defaultSentryClientGetter()
+    public static SentryClient defaultSentryClientGetter(final String dsn)
     {
-        return SentryClientFactory.sentryClient();
+        return Sentry.init(dsn);
```

Comment:
Is it better to keep this instance as singleton at the first time and to return it when this method is called next time?

Reply:
FYI : According source code in Sentry, you can write like `SentryClientFactory.sentryClient(dsn, null)`

```
    public static SentryClient init(String dsn, SentryClientFactory sentryClientFactory) {
        SentryClient sentryClient = SentryClientFactory.sentryClient(dsn, sentryClientFactory);
        setStoredClient(sentryClient);
        return sentryClient;
    }
```

Reply:
`setupSentry` should be called at just once in processes in normal processes, or twice or more in tests. For both cases, implicit singleton objects may cause troubles.